pkg/ingestor/parser/spdx/parse_spdx.go

```diff
@@ -65,6 +70,14 @@ func (s *spdxParser) Parse(ctx context.Context, doc *processor.Document) error {
 		return fmt.Errorf("failed to parse SPDX document: %w", err)
 	}
 	s.spdxDoc = spdxDoc
+	if spdxDoc.CreationInfo == nil {
```

Comment:
qq: is this a required field by SPDX?

Reply:
yes, I was hesitant to add, but looks required: https://spdx.github.io/spdx-spec/v2-draft/document-creation-information/#69-created-field